docker-entrypoint.sh

```diff
@@ -365,6 +395,9 @@ docker_setup_db() {
 		fi
 	fi
 
+	# To create replica user
+	create_replica_user
+
```

Comment:
I'm expecting some form, directly or function, that has:
```
if [ "$MARIADB_REPLICATION_MODE" = "replica"

   CHANGE MASTER TO .. MASTER_CONNECT_RETRY=10;
   START REPLICA;
```
In this case only `MARIADB_REPLICATION_PASSWORD` is supported and not the hash.

That is executed. I don't think I need to poll the master to ensure its up before `start replica`, retry interval can be used.